moduleframework/helpers/openshift_helper.py

```diff
@@ -61,7 +62,7 @@ def __init__(self):
         random_str = ''.join(random.choice(string.lowercase) for _ in range(3))
         self.project_name = "{project}-{random_str}".format(project=self.app_name,
                                                             random_str=random_str)
-        common.print_debug(self.icontainer, self.app_name)
+        print_debug(self.icontainer, self.app_name)
```

Comment:
Any idea, why you remove common? I thing, `common` means from which module `print_debug` is loaded.

Reply:
because when using module common there, it hides errors when some bigger change, thats why I tries in more cases use these from common import ... things, what shows me, something what I've changed significantly (caused by this rewriting to use mtf config file)